Starcounter.Uniform/ViewModels/UniDataTable.json.cs

```diff
@@ -142,11 +145,29 @@ public void Handle(Input.PageSize action)
         [UniDataTable_json.Columns]
         public partial class ColumnsViewModel : Json, IBound<DataTableColumn>
         {
+            private string _sortValue = null;
+
             public IFilteredDataProvider<Json> DataProvider { get; set; }
             public Action LoadRowsFromFirstPage { get; set; }
 
-            private static string Descending => "desc";
-            private static string Ascending => "asc";
+            /// <summary>
+            /// Empty string is also a "value" on the client side, which results into invalid HTML attribute:
```

Comment:
Please fix the summary comment:
![image](https://user-images.githubusercontent.com/25042256/55789257-40daaa80-5aba-11e9-9c81-07a0ca0d1804.png)